hono/src/services/outfits.ts

```diff
@@ -33,94 +33,91 @@ const selectOutfitSchema = createSelectSchema(outfits, {
 })
 
 app.get('/', injectDB, async (c) => {
-  return c.json(
-    await c.get('db').query.outfits.findMany({
-      with: {
-        itemsToOutfits: {
-          columns: {
-            itemId: false,
-            outfitId: false,
-          },
-          with: {
-            item: true,
-          },
-          orderBy: (itemsToOutfits, { asc }) => [asc(itemsToOutfits.itemType)],
+  const res = await c.get('db').query.outfits.findMany({
+    with: {
+      itemsToOutfits: {
+        columns: {
+          itemId: false,
+          outfitId: false,
         },
+        with: {
```

Comment:
Last min nit: I would suggest exploring if we can include the itemsExtended materialized view in the array of items that are returned per outfit. This would be incredibly useful in the frontend as the component that displays the outfit will need the same attributes as itemsExtended.

Reply:
Attempts to integrate materialized view for items:

- using `extra` keyword to retrieve attributes from `itemsExtended` (after using `items` relation)
- using `where` keyword to filter items from `itemsExtended` view by comparing ids from `items` table (need to do this because `itemsExtended` view does not have a relation to `itemsToOutfits`)

Reply:
I've spent a bit of time thinking on this, and I believe a better approach would:
- NOT use materialized views, instead we can just create a regular view that is not cached
- Caching could then be done by a separate technology that is designed to do such a thing (Redis, Cloudflare Hyperdrive)
Once this is modified into a regular view, we can tackle how to return items as apart of the outfits GET API endpoint.